index.html

```diff
@@ -1,29 +1,136 @@
+     <p> Bacon ipsum dolor amet frankfurter porchetta kielbasa <br>
+        turkey jowl prosciutto boudin short loin burgdoggen sausage <br>
+        shank drumstick. Chuck strip steak leberkas pastrami beef <br>
+        ribs burgdoggen landjaeger short loin ham frankfurter brisket. <br>
+        Tenderloin swine t-bone, meatloaf hamburger chicken pork alcatra <br>
+        ball tip prosciutto. Bacon venison cupimp beef ribs.</p>
+
+        <button type="button"> More Info</button>   
+</div>
+
+ <!--This class outlines the text that describes this weeks challenge -->
+<div class= challenge>
+   <h1> Upcoming Event: <br> Hot Wing Wednesday <br> This Week: <br> 
+   10 Hot Wings With Suicide Sauce </h1>
+</div>
+
+ <!--This class outlines the image of the food from this weeks challenge as well as the heading and paragraph -->
+<div class= wings>
+   <img width = "500" src="sauce_and_wings.png" alt="image of sauce and wings"> 
+   <h2> Cash Prize: $200</h2>
+   <p> Bacon ipsum dolor amet frankfurter porchetta kielbasa <br>
+      turkey jowl prosciutto boudin short loin burgdoggen sausage <br>
+      shank drumstick. Chuck strip steak leberkas pastrami beef <br>
+      ribs burgdoggen landjaeger short loin ham frankfurter brisket. <br>
+      Tenderloin swine t-bone, meatloaf hamburger chicken pork alcatra <br>
+      ball tip prosciutto. Bacon venison cupimp beef ribs.</p>
+
+</div>
+
+ <!--This class outlines the map image as well as the resturant's hours and address-->
+<div class = map> 
+   <img width = "450" src="map_3.png" alt="image of restaurant on map"> 
+   <h1> Open Monday-Friday <br> 12am-8pm <br> 1245 Spice Rd. N, <br> Maple Leaf, <br> Ontario </h1>
+   <button type="button"> Find Other Locations</button>
+</div>
 
-    <!-- We are using a CSS Reset file to standardize browser default css -->
-    <link rel="stylesheet" href="reset.css" />
+ <!--This class outlines the text that reads 'Subscribe to Our Newsletter -->
+<div class = subscribe> 
+<h1> Subscribe To Our NewsLetter: </h1>
+</div>
 
-    <!-- This is where all of our awesome CSS code goes -->
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <!--
+ <!--This form allows the user to input their name, and their email, 
+   there is also a submit button and a reset button -->
+<div class = newsletter> 
+<form>
+   <label for="Name"> Name: </label> 
+   
+   <input type="text" id="Name" name="your name" autocomplete = "name" placeholder="Your Name..."> <br> <br>
+   <label for="Email"> Email: </label> 
+   
+   <input type = "text" id="Email" name="email" autocomplete = "email" placeholder="Your Email...">
+   <button type="button">Sign Up</button>
+</form>
+</div>
 
-      YOU HAVE TO MODIFY THE HTML TO BUILD YOUR AWESOME PAGE
+     <div class = foot> 
+     <img width="100" src="logo_v3.png" alt="footer logo">
+     
+     <h1 style="color:aliceblue"> PEPPER PETE'S</h1>
+     <p style="color:aliceblue"> © 2023 Pepper Pete's </p>
+   
```

Comment:
You should only have one `<h1>` on your page. 

The other section headers can start at an `<h2>`